packages/react-instantsearch/src/connectors/connectCurrentRefinements.js

```diff
@@ -1,17 +1,22 @@
 import createConnector from '../core/createConnector';
-
+import {PropTypes} from 'react';
 /**
  * connectCurrentRefinements connector provides the logic to build a widget that will
  * give the user the ability to remove all or some of the filters that were
  * set.
  * @name connectCurrentRefinements
  * @kind connector
+ * @propType {function} transformItems - If provided, this function can be used to modified the `items` provided prop of the wrapped component (ex: for filtering or sorting items). this function take the `items` prop as a parameter and expect it back in return.
```

Comment:
That's a bug in the way it is parsed then :) The thing is that there is an implicit default value, identity